exopy_hqc_legacy/tasks/tasks/instr/psa_tasks.py

```diff
@@ -23,10 +22,10 @@
 
 class PSAGetTrace(InstrumentTask):
     """ Get the trace displayed on the Power Spectrum Analyzer.
-
     """
     trace = Int(1).tag(pref=True)
-
+    mode = 'SA'
```

Comment:
Do you really a class variable ? A `atom.Enum` seems like a good fit here.

Reply:
I replaced this by:
mode  = Enum('SA').tag(pref=True)
I have no idea how this syntax works but based on observation it should be the right one